tools/clang/test/CodeGenHLSL/batch/declarations/functions/entrypoints/semantics/cs_sv_id_not_int32.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -E main -T cs_6_2 -enable-16bit-types %s | FileCheck %s
```

Comment:
It would be nice to test the min-precision case as well, since that was the original bug.  Also, do we error on float or bool types elsewhere?

Reply:
Added test, and filed #2299